Update the contribution requirement (#853)
OpenLiberty/guides-common: CONTRIBUTING.md

## Contributions

> Contributing to an Open Liberty guide is simple. All you need to do is fork from the `prod` branch, create a branch in your forked repository to include your contribution, and open a pull request to the `staging` (quality assurance) branch for the particular guide. You need to certify the originality of the work following the [Developer Certificate of Origin (DCO)](https://developercertificate.org) approach by adding a line to the end of the Git commit message to sign your work:

Reviewer: ```suggestion
Contributing to an Open Liberty guide is simple. Each guide is maintained in its own GitHub repository. To contribute, just fork from the `prod` branch for your chosen guide.  Then create a branch in your forked repository to include your contribution and open a pull request to the `staging` branch for the guide. Certify the originality of your work by following the [Developer Certificate of Origin (DCO)](https://developercertificate.org) approach and adding a line to the end of the Git commit message to sign your work:
```

Many Git UI tools have support for adding the `Signed-off-by` line to the end of your commit message. This line can be automatically added by the `git commit` command by using the `-s` option.

> The team will be notified for review. Following will be the review process:

Reviewer: ```suggestion
The team is then notified and your contribution is reviewed according to the following process:
```

1. The team will review your change(s).
    - If there are further changes to be made, the team will request changes on the pull request.
    - If the team does not agree with the change, the PR will be closed with an explanation and suggestion for follow-up.
2. If the team approves, the team will merge your PR. A full Open Liberty staging site build will be run.
3. Based on the results of the build:
    - If further review is needed, we will let you know about a pending review from our team and discuss any necessary improvements that need to be made to your change(s).

>     - If everything is successful, the team will publish your change(s) to the `prod` branch, and update the Open Liberty production site if necessary.

Reviewer: ```suggestion
    - If everything is successful, the team will publish your change(s) to the `prod` branch and update the Open Liberty production site, if necessary.
```
